consultation_analyser/support_console/views/consultations.py

```diff
@@ -46,15 +46,21 @@ def delete(request: HttpRequest, consultation_id: UUID) -> HttpResponse:
     return render(request, "support_console/consultations/delete.html", context=context)
 
 
-def get_number_themes_latest_run(consultation):
+def get_number_themes_for_processing_run(consultation, processing_run=None):
+    """
+    Get total themes for processing run.
+    If processing_run not specified, get the latest processing run if exists.
```

Comment:
This is a bit magic I think. Better to resolve processing_run once in the view method and have it be either present or `None` here instead of silently fetching?

Reply:
Yes, have now updated to just take a processing_run, which we pass in from the view (at the moment, just the latest processing run, as we haven't made changes to the user-facing frontend).